modules/ROOT/pages/values-and-types/temporal.adoc

```diff
@@ -527,6 +527,7 @@ Parsing a `ZONED DATETIME` using the _calendar date_ format:
 RETURN datetime('2015-06-24T12:50:35.556+0100') AS theDateTime
 ----
 
+[caption=]
```

Comment:
What do these do?

Reply:
they remove the "Table x" enumeration from the result tables in the examples.
they're page-global and i noticed that since there is a table before the examples, the indexes of the example tables are offset by 1 relative to the numbering of the examples.

![Screenshot from 2024-02-29 08-45-26](https://github.com/neo4j/docs-cypher/assets/156673635/675eaaa3-1b9c-4357-9845-774f55aea0ec)

vs

![Screenshot from 2024-02-29 08-44-56](https://github.com/neo4j/docs-cypher/assets/156673635/76f433b3-7412-4d34-b6a7-65cc6a277e36)

Reply:
thought of that too, but then decided to rather get rid of the result table numbering because the examples are already numbered

Reply:
disabled them in both files.